### PR TITLE
Add age and mysql

### PR DESCRIPTION
**Pull request by hw2cb**
Please, review

### CODE REVIEW

File: mvnw.cmd [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @REM regarding copyright ownership.  The ASF licenses this file
 @REM to you under the Apache License, Version 2.0 (the
 @REM "License"); you may not use this file except in compliance
-@REM with the License.  You may obtain a copy of the License at
+@REM with the License.  You may obtain a copy of the License at33
```
Comment (Dau-Nugmanov) [inline mvnw.cmd]: Revert

File: src/main/resources/templates/owners/ownerDetails.html [hunk 2]
```diff
@@ -26,6 +26,10 @@ <h2>Owner Information</h2>
         <th>Telephone</th>
         <td th:text="*{telephone}"></td>
       </tr>
+      <tr>
+        <th>Возраст</th>
```
Comment (Dau-Nugmanov) [inline src/main/resources/templates/owners/ownerDetails.html]: Rename to Age

File: src/main/resources/templates/owners/ownersList.html [hunk 3]
```diff
@@ -14,6 +14,7 @@ <h2>Owners</h2>
     <th>City</th>
     <th style="width: 120px">Telephone</th>
     <th>Pets</th>
+    <th>Возраст</th>
```
Comment (Dau-Nugmanov) [inline src/main/resources/templates/owners/ownersList.html]: Rename to Age

File: src/main/resources/templates/vets/vetList.html [hunk 4]
```diff
@@ -12,6 +12,7 @@ <h2>Veterinarians</h2>
   <tr>
     <th>Name</th>
     <th>Specialties</th>
+    <th>Возраст</th>
```
Comment (Dau-Nugmanov) [inline src/main/resources/templates/vets/vetList.html]: Rename to Age

**Review by Dau-Nugmanov — COMMENTED** [review]
Please fix my comments

**Comment by hw2cb** [comment]
> Please fix my comments

Completed

**Review by Dau-Nugmanov — APPROVED** [review]

**Pull request merged**